cfme/tests/networks/nuage/test_events_are_triggered.py

```diff
@@ -61,6 +65,53 @@ def with_nuage_sandbox(networks_provider):
     logger.info('Destroyed sandbox enterprise {} ({})'.format(enterprise.name, enterprise.id))
 
 
+@pytest.fixture
+def targeted_refresh(merkyl_inspector):
+    """
+    This fixture tests whether targeted refresh was triggered for given targets.
+    It basically tails evm.log to see whether a line like this has occured:
+
+    ```
+    [Collection of targets with id: [{:ems_ref=>"b35d3afe-6f19-4da8-b1ee-b79de433cace"}, ... ]]
+    ```
+    for each target and raises an error if not.
+
+    Usage:
+
+        def test_something(targeted_refresh)
+                trigger_targeted_refresh() <- some function that will trigger
+                                              targeted refresh
+                with targeted_refresh.target_timeout():
+                    targeted_refresh.register_target('ref-123456', 'Subnet named TEST')
+                    targeted_refresh.register_target('ref-aaaabb', 'Router named TEST')
+    """
+    evm_log = '/var/www/miq/vmdb/log/evm.log'
+    needle_template = r'^.*Collection of targets with id.*:ems_ref=>"{}".*.$'
+    merkyl_inspector.add_log(evm_log)
+    targets = set()  # { (ems_ref, comment), (ems_ref, comment), ... }
+
+    def check_log():
+        for target in set(targets):
```

Comment:
Please add following line right before the `for` loop

```
logger.info('Looking for %s needles in evm.log: %s', len(targets), [t[1] for t in targets])
```

in order for user to be able to follow the progress when the test runs.